editor/plugins/org.fusesource.ide.camel.editor/src/org/fusesource/ide/camel/editor/globalconfiguration/beans/wizards/pages/GlobalBeanEditWizardPage.java

```diff
@@ -127,76 +89,18 @@ protected void doSetValue(Object value) {
 				}
 				getValue();
 			}
-		};
+		};		
```

Comment:
optional: tabs has been added at the end of the line, would be better to remove them